news/2018-09-25-project-loved-one-year-week-of-september-23rd.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+title: "Project Loved: One Year / Week of September 23rd"
+date: 2018-09-25 20:00:00 +0000
+---
+
+[Project Loved](https://osu.ppy.sh/community/forums/120) is back again with a wide selection of beatmaps from all modes. Don't forget to cast your votes! We're also celebrating our first birthday today, so show us some love too!
+
+[![](/wiki/shared/news/banners/loved.png)](https://osu.ppy.sh/community/forums/120)
+
+"[Introducing the Loved Captain's Pick](https://osu.ppy.sh/home/news/2017-09-25-introducing-the-loved-captains-pick)"—remember this announcement? Can you believe that it was made an entire year ago? [Project Loved](https://osu.ppy.sh/community/forums/120) is celebrating its first birthday under the "Captain's Pick" system, and we're approaching two years since [the Loved category was brought to life](https://osu.ppy.sh/home/news/2016-10-17-here-comes-the-love). Loved has had some huge success in bringing leaderboards and recognition to the beatmaps that stand out to the community, and it's all been possible thanks to the people who've helped out. So, before we get into the meat of this week's post, I want to thank everyone that's been a part of this project:
```

Comment:
Paragraph break after "...brought to life."? This textblock stands out as being quite big.

Reply:
Also might be my ignorance but is contracting to simply "Loved" fine? Not sure if you want to continue referring to it as "Project Loved" or "The Loved Category".